### PR TITLE
Updated ios leo

### PR DESCRIPTION
**Pull request by witmicko**
including previously closed IOS fix

### CODE REVIEW

**Comment by jcesarmobile** [comment]
This is installing cordova@6.3.1, we should remove that

**Comment by odra** [comment]
you can replace all instances of "node_modules/cordova/bin/cordova" to "codova" (to use global cordova)

File: Jenkinsfile [hunk 1]
```diff
@@ -28,19 +41,40 @@ node(platform) {
...
     }
 
     stage("Build") {
-        withEnv(['CORDOVA_ANDROID_GRADLE_DISTRIBUTION_URL=https://services.gradle.org/distributions/gradle-2.13-bin.zip']) {
-            if (BUILD_CONFIG == 'debug') {
-               sh 'node_modules/cordova/bin/cordova build $platform --debug'
-            } else {
-               sh 'node_modules/cordova/bin/cordova build $platform --release'
+        if (platform == 'android') {
+            withEnv(['CORDOVA_ANDROID_GRADLE_DISTRIBUTION_URL=https://services.gradle.org/distributions/gradle-2.13-bin.zip']) {
```
Comment (odra) [inline Jenkinsfile]: please remove the withEnv directive (we do not need it anymore)

**Comment by odra** [comment]
LGTM

**Pull request merged**